src/instrumenter/instrument.ts

```diff
@@ -127,6 +134,26 @@ const CHECK_STATE_INVS_FUN = "__scribble_check_state_invariants";
 const OUT_OF_CONTRACT = "__scribble_out_of_contract";
 const CHECK_INVS_AT_END = "__scribble_check_invs_at_end";
 
+function getAllNames(contract: ContractDefinition): Set<string> {
```

Comment:
I think this function might not include names implicitly included as part of an import. E.g. in the below example the enum definition `vars0` from `lib.sol` is implicitly shadowed in `foo.sol` which imports `lib.sol`.

lib.sol

enum vars0 {
  A,B,C
}

foo.sol:
import "lib.sol"
contract Foo {
   struct vars0 {...};
   function foo() {
      vars0 _v;
      ...
   }
}